shor/layers.py

```diff
@@ -30,5 +30,23 @@ def __getitem__(self, key):
         return self._qbits[key]
 
 
+class Cbits(_Layer, Iterable):
```

Comment:
This looks good. We will later want to change the _cbits to be a tuple of tuples `tuple((idx, self) for idx in range(num))`

But don't need to yet I believe. (If we only need a single qbit and cbit register for the conditional measurement)

Reply:
OKay Ill hold off on updating this